guides/source/upgrading_ruby_on_rails.md

```diff
@@ -22,7 +22,7 @@ Rails generally stays close to the latest released Ruby version when it's releas
 
 * Rails 7 requires Ruby 2.7.0 or newer.
 * Rails 6 requires Ruby 2.5.0 or newer.
-* Rails 5 requires Ruby 2.2.2 or newer.
+* Rails 5 requires Ruby 2.2.2 through 2.6.10.
```

Comment:
I think we should just remove this line instead of updating it, Rails 5 is no longer supported.

Reply:
This is the document that describes how to move to a supported version. See https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-5-2-to-rails-6-0